tests/py3_driver_testers.txt

```diff
@@ -257,6 +257,11 @@ Retevis_RT9000D_400-490,@KC9HI,8-Dec-2022
 Retevis_RT9000D_66-88,@KC9HI,8-Dec-2022
 TDXone_TD-Q8A,@KC9HI,18-Dec-2022
 TIDRADIO_TD-H6,@kk7ds,21-Oct-2022
+TIDRADIO_TD-H8,@Sandmann34,7-Feb-2023
+TIDRADIO_TD-H8-GMRS,@Sandmann34,7-Feb-2023
```

Comment:
You should also include a test image for this one because of the special memory behaviors, and probably the ham variant as well. Since it looks like the UV68 is identical to the TD-H8, no need to  have one for that.

Reply:
Do you mean that I need to reposition the image of tdh8 and add the test image of ham and gmrs? Because I placed one in images before.

Reply:
Just add the ham and gmrs images to what you have here. The PR here only adds one image (see "Files changed" tab), perhaps you had others in a previous rev? But yeah, just add those two in addition to what you have here.